### DISCUSSION TITLE
Simple script to display JWST and SEMB-L2 using Nasa Horizons position information

### DISCUSSION THREAD

**Discussion by nbhasker** [Stellarium/stellarium#2183] [Show and tell]
Hi - I've written a simple Stellarium script to display the position of the JWST spacecraft and the Sun-Earth L2 point on multiple days. There's no magic - data has to be manually imported from Nasa Horizons. See https://github.com/nbhasker/JWSTStellarium

Lots of folks (including me) are fascinated by the halo orbit planned for JWST and this may help us visualize it better in the coming weeks when the spacecraft enters its orbit around L2.

Best Regards

Bhasker

**Comment by Keatah** [comment]
bad link

**Reply by nbhasker** [reply]
Sorry for the fumble. Here's the link - https://github.com/nbhasker/JWSTStellarium.
